tests/default_vars.sh

```diff
@@ -1186,7 +1186,7 @@ export USE_MERRA2=.true.
 export IAER=1011
 export NSTF_NAME=2,1,0,0,0
 
-export LHEATSTRG=.true.
```

Comment:
Changing the default values may change the results of other tests. You need to run the full regression test suite for Intel and GNU to confirm results do not change.